config/jobs/ci-infra/build-golang-test-image.yaml

```diff
@@ -8,6 +8,7 @@ postsubmits:
     annotations:
```

Comment:
The problem here is: we have to trigger this job manually, if we only change the build arg value in the job config but nothing else. 
We might need to consider putting tags and args into a file under `/images` or something similar to what test-infra does: https://github.com/kubernetes/test-infra/tree/master/images/bazel
But we can run with this for now, I guess.